utils.js

```diff
@@ -43,4 +43,53 @@ function _in(needle, haystack) {
     return false;
 }
 
-module.exports = {parsePrice, composeRanges, _in};
\ No newline at end of file
+function formatHeader(player, auctionMode) {
+
+    let arrayBuffer = [];
+    let padding = ''
+    //70 for iphone, 124 for desktop
+    for (let i = 0; i < 70 - (player.length + auctionMode.length); i++){
+        arrayBuffer.push(' ');
+    }
+    if (auctionMode === '---') {
+        console.log('---')
+        padding = '     ';
+    }
+    const test = `${player}${arrayBuffer.join('')}${padding}*${auctionMode}*`
+    console.log(test.length)
+    return `${player}${arrayBuffer.join('')}${padding}*${auctionMode}*`;
+}
+
+function formatCapitalCase(sentence) {
```

Comment:
I believe if you look up the item in the items.json  in all-caps by key, you can take the "wiki URL" (value) and replace underscores with spaces, and you have the correct capitalization?

Reply:
I use this for most everything, like prices and server names too.

Reply:
Ah, interesting... Ok. Might be fine, but IMO would be better then to switch the specific locations where we know it's item names to use the method I'm talking about, rather than using this function. But, maybe for display purposes it's just "good enough".